tools/c7n_gcp/c7n_gcp/filters/essentialcontacts.py

```diff
@@ -0,0 +1,62 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+"""
+Essential Contacts suppport for GCP organization
+"""
+from c7n.filters.core import ValueFilter
+from c7n.utils import local_session, type_schema
+from c7n_gcp.resources.resourcemanager import Organization
+import jmespath
+
+
+@Organization.filter_registry.register('essentialcontacts')
```

Comment:
general pref to hyphenate on compound words.

Reply:
Thx, made the filter name "essential-contacts" now.